libs/partners/fireworks/pyproject.toml

```diff
@@ -0,0 +1,96 @@
+[tool.poetry]
+name = "langchain-fireworks"
+version = "0.0.1"
+description = "An integration package connecting Fireworks and LangChain"
+authors = []
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain"
+license = "MIT"
+
+[tool.poetry.urls]
+"Source Code" = "https://github.com/langchain-ai/langchain/tree/master/libs/partners/fireworks"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+langchain-core = "^0.1"
+fireworks-ai= "^0.12.0"
```

Comment:
just confirming we want to exclude a 0.13.0 release. Otherwise might make sense to update to `>=0.12.0,<1`